packages/cli-kit/src/private/node/conf-store.ts

```diff
@@ -89,3 +91,21 @@ export async function cacheRetrieveOrRepopulate(
   config.set('cache', cache)
   return value
 }
+
+/**
+ * Fetch from cache if already populated, otherwise return undefined.
+ * @param key - The key to use for the cache.
+ * @returns The value from the cache or the result of the function.
+ */
+export function cacheRetrieve(
+  key: keyof Cache,
+  config = cliKitStore(),
+): CacheValueForKey<typeof key> | undefined {
+  const cache: Cache = config.get('cache') || {}
+  const cached = cache[key]
+  return cached?.value
+}
+
+export function cacheClear(config = cliKitStore()): void {
```

Comment:
Are we using this?

Reply:
Yes, it is used for tests to reset cache state.

It's not used in `conf-store.test.ts` itself because we can always inject a conf store. But for higher-level tests, I think it makes sense to use the real thing and reset the cache between tests.